src/util/saveCurrentCursorOffset.js

```diff
@@ -0,0 +1,10 @@
+import debounce from 'lodash.debounce'
+
+const saveCurrentOffsetToLocalStorage = debounce(() => {
+  const offset = window.getSelection().focusOffset
+  localStorage.setItem('currentCursorPosition', offset)
+}, 10)
+
+export default function saveCurrentCursorOffset() {
+  saveCurrentOffsetToLocalStorage()
+}
```

Comment:
I would suggest defining and exporting a single function in this file. No need to have two functions if you are `debouncing` inline.

Reply:
I was thinking maybe there'd be something to do there at some point in time.
But maybe yeah, it was overthinking. I'll just export one function